04-Multi-Band-Raster.Rmd

```diff
@@ -70,22 +65,36 @@ Read more about the `raster` package in R.</a>
 
 #About Raster Bands
 
-As discussed in the [Intro to Raster Data Lesson 00]( {{ base.url }} }}/R/Introduction-to-Raster-Data-In-R), a raster can contain 1 or more bands. To 
-work with multi-band rasters, we need to adjust how we import and plot our data. 
+As discussed in the [Intro to Raster Data Lesson 00]( {{ base.url }} }}NEON-R-Spatial-Raster/R/Introduction-to-Raster-Data-In-R), a raster can
+contain 1 or more bands. To work with multi-band rasters, we need to change how 
+we import and plot our data in several ways. 
 
-* To import multi band raster data we will use the `stack` function.
-* If our multi-band data are imagery that we wish to composit, we can use `plotRGB` 
-to plot a 3 band image raster.
+* To import multi band raster data we will use the `stack()` function.
+* If our multi-band data are imagery that we wish to composite (spatial data
+term essentially meaning, to view multiple bands together), we can use
+`plotRGB()` to plot a 3 band image raster.
+
+>Bonus: Bands? Layers?  Why are both names used? The term bands comes from the 
```

Comment:
This explanation could go in an earlier lesson (00?), however, this was the first lesson that started to really use the term layers instead of bands so it may belong here.